js/src/dapps/registry/accounts/actions.js

```diff
@@ -0,0 +1,23 @@
+const { personal } = window.parity.api;
+
+export const set = (accounts) => ({ type: 'accounts set', accounts });
+
+export const fetch = () => (dispatch) => {
```

Comment:
IMHO all async actions should always consist of three sub-actions: `fetch_started, fetch_success, fetch_error`